src/misc.jl

```diff
@@ -97,15 +124,30 @@ function findat!{T}(r::IntegerArray, a::AbstractArray{T}, b::AbstractArray{T})
     return r
 end
 
+
+"""
+    findat(a, b) -> Array
```

Comment:
"-> Vector"

(This function should be deprecated in favor if `indexin` from Base, but that's for another PR.)